### PR TITLE
Rocket setup

### PR DESCRIPTION
**Pull request by arabiu033**
## Changes made :star: 

- Structure the Rocket module
- Fetch the Rockets' content from API
- Display The fetch in the requirement structure
- Add some small styling for display
- Fix Linters

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
+const logo = require('../assets/images/planet.png');
+
```
Comment (ndikumanaisaie) [inline src/components/Header.jsx]: - Kindly use the ES6 module instead of the commonjs module.`import logo from ....`

**Review by ndikumanaisaie — CHANGES_REQUESTED** [review]
### Hi @arabiu033 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

✅ Descriptive pull request message
✅ linter error checks passing
✅ Documented your work professionally
✅ Descriptive commit message.

However, there are a few things that you still have to work on. Check the comments below!

File: src/components/Header.jsx (hunk 1 above)
Comment (arabiu033) [reply src/components/Header.jsx]: Done :+1:

**Review by ndikumanaisaie — APPROVED** [review]
# APPROVED!!!  🎉 🎉 🎉 
### Hi @arabiu033 

### Good job !!!
### To Highlight!

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Descriptive commit messages

Congratulations! 🎉 🎉 🎉

There is nothing to add rather then it is time to merge.

**Pull request merged**